CSharp/DevVmPowershell/Helpers/Implementations/InstanceSettingsHelper.cs

```diff
@@ -87,9 +152,12 @@ public void DeleteInstanceSetting(int instanceSettingArtifactId)
 		{
 			try
 			{
-				using (IInstanceSettingManager instanceSettingManager = ServiceFactory.CreateProxy<IInstanceSettingManager>())
+				HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+				string deleteUrl = $"Relativity.REST/api/Relativity.InstanceSettings/workspace/{Constants.EDDS_WORKSPACE_ARTIFACT_ID}/instancesettings/{instanceSettingArtifactId}";
```

Comment:
I would move all the URL's to the Constants class so they are in one place for easy updates in future.

- [ ] Move URL to Constants class